modules/apps/object/object-api/src/main/java/com/liferay/object/service/ObjectDefinitionLocalServiceWrapper.java

```diff
@@ -112,14 +112,6 @@ public com.liferay.object.model.ObjectDefinition addSystemObjectDefinition(
 			objectFields);
 	}
 
-	@Override
-	public void bindObjectDefinitions(long[] objectRelationshipIds)
-		throws com.liferay.portal.kernel.exception.PortalException {
-
-		_objectDefinitionLocalService.bindObjectDefinitions(
-			objectRelationshipIds);
-	}
```

Comment:
Hey @victorKammerer please move this change to the Build Service commit.

Thank you!

Reply:
Done, Thanks!